src/js/common/schemaform/validation.js

```diff
@@ -87,17 +87,33 @@ export function transformErrors(errors, uiSchema) {
  * This pulls custom validations specified in the uiSchema and validates the formData
  * against them.
  *
- * Expects validations that look like:
+ * Expects validations that EITHER look like:
  *
  * someField: {
  *   "ui:validations": [
  *     someValidationFunction,
  *   ]
  * }
  *
- * The function is passed errors, fieldData, pageData, formData, and otherData and
- * should call addError to add the error.
- *
+ * with each validation function having a signature of:
+ * validation(pathErrors, currentData, formData, schema, uiSchema['ui:errorMessages'], currentIndex)
```

Comment:
These signatures are different...looks like the last change was over a year ago. Oh! By me. Yeah, it's definitely outdated by now. Thanks for updating this!

Can you actually make this consistent with what's in the form config doc I linked to above?